src/main/webapp/app/layouts/navbar/navbar.component.ts

```diff
@@ -78,13 +78,11 @@ export class NavbarComponent implements OnInit, OnDestroy {
      * Check if a guided tour is available for the current route to display the start tour button in the account menu
      */
     subscribeForGuidedTourAvailability(): void {
-        this.router.events.subscribe((event: Event) => {
-            if (event instanceof NavigationEnd) {
-                this.isTourAvailable = this.guidedTourService.checkGuidedTourAvailabilityForCurrentRoute();
-            }
-        });
         // Check availability after first subscribe call since the router event been triggered already
-        this.isTourAvailable = this.guidedTourService.checkGuidedTourAvailabilityForCurrentRoute();
+        this.guidedTourService.getGuidedTourAvailabilityStream().subscribe(isAvailable => {
+            this.isTourAvailable = isAvailable;
+            console.log('tour available: ', this.isTourAvailable);
```

Comment:
I think we should only do `console.log` on errors.

Reply:
The log was only used for testing - I missed it when I was removing my console.logs. I will remove it now :)